src/Apps/__tests__/Fixtures/Artwork/ArtworkSidebar/ArtworkSidebarBidAction.ts

```diff
@@ -106,7 +106,7 @@ export const UserPendingIDV: ArtworkSidebarBidAction_Test_Query$rawResponse["me"
 export const IDVedUser: ArtworkSidebarBidAction_Test_Query$rawResponse["me"] = {
   id: "user-id",
   isIdentityVerified: true,
-  pendingIdentityVerification: undefined,
+  pendingIdentityVerification: null,
 }
```

Comment:
These are fixes are for typescript error that was showing up unrelated to our work, let me know if it makes sense.

Reply:
Seems that undefined works after rebase - will revert this.